Cargo.toml

```diff
@@ -115,7 +116,6 @@ starknet_api = { git = "https://github.com/starkware-libs/starknet-api.git", tag
 [dev-dependencies]
 rstest = { version = "0.18.1", default-features = false }
 toml = { version = "0.7.5", default-features = false }
-rand = { version = "0.8.5", default-features = false }
```

Comment:
I guess it can't be in `dev` because we need it in `testing` feature? Can you set it as optional and add it to the `testing` and `arbitrary` feature?